mp/importer/tests/test_batcher.py

```diff
@@ -0,0 +1,124 @@
+import os
+from unittest import TestCase
+import collections
+from collections import namedtuple
+from collections import Iterable
+
+def make_test_context_1():									# creates a "provisional" context 
+    context = namedtuple('Context', """
```

Comment:
why not pass the parameters in as defaults?
like:

def make_test_context(batch_size=2, start_batch=0):

then when you need different values, you can call it like: make_test_context(start_batch=2) rather than make_test_context_2()